bgpd/bgp_bmp.c

```diff
@@ -746,6 +747,32 @@ static int bmp_peer_backward(struct peer *peer)
 	return 0;
 }
 
+static int bmp_peer_deleted(struct peer *peer)
+{
+	if (peer->status != Deleted) {
+		return 0;
+	}
+
+	if (BGP_DEBUG(bmp, BMP)) {
```

Comment:
No curling braces for a single statement, please. Also, please use `__func__` instead of hard-coded function names.